### PR TITLE
feature: add helper to check if all required labels are available

### PR DESCRIPTION
**Pull request by sumitanvekar**
### :pencil: Description
add helper to check if all required labels are available

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -79,6 +79,9 @@ inputs:
   number_of_assignees:
     description: 'Number of people to assign a PR to'
     required: false
+    contains_all_labels:
+    description: 'A comma seperated list of labels used to check if all of them are present on a pull request'
+    required: false
```
Comment (danadajian) [inline action.yml:84]: We should be able to use the existing `labels` input here.

**Review by danadajian — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (sumitanvekar) [reply action.yml:84]: i kept it separate since it gives more option in future. e.g if we want to add checks like "one of", "none of", etc.

Comment (danadajian) [reply action.yml:84]: Actually, I just realized this can be accomplished within github workflows already

Comment (danadajian) [reply action.yml:84]: https://docs.github.com/en/actions/learn-github-actions/expressions#contains

**Comment by sumitanvekar** [comment]
I am closing this pr since it can be achieve with native gh actions support. Thanks Dan!

**Pull request closed without merging**